syntax = "proto3";

package rules;

message MomentumRequest {
    string symbol = 1;
    int32 lfast = 2;
    int32 lslow = 3;
    bool use_attenuation = 4;
    float scaling_factor = 5;
    string scaling_type = 6;

}

message MomentumResponse {
    bytes series = 1;
}

service Momentum{
    rpc get_momentum (MomentumRequest) returns (MomentumResponse);
}